readme.md

```diff
@@ -285,6 +285,7 @@
 - [Material-UI](https://github.com/nadunindunil/awesome-material-ui#readme) - Material Design React components for faster and easier web development.
 - [Building Blocks for Web Apps](https://github.com/componently-com/awesome-building-blocks-for-web-apps#readme) - Standalone features to be integrated into web apps.
 - [Svelte](https://github.com/TheComputerM/awesome-svelte#readme) - App framework.
+- [Design systems](https://github.com/klaufel/awesome-design-systems#readme) - An ever evolving collection of reusable components, guided by rules that ensure consistency and speed, by being the single source of truth for any product development.
```

Comment:
Would be great if you could shorten the description slightly.

Reply:
Updated, now the description is on one line.